src/coreComponents/codingUtilities/tests/testTable.cpp

```diff
@@ -0,0 +1,393 @@
+             "|  Duis fringilla, ligula sed porta fringilla, ligula wisi commodo felis,ut adipiscing felis dui in enim. Suspendisse malesuada ultrices ante  |  [30.21543]  |  30.45465142  |  787442   |    10     |\n"
+             "+----------------------------------------------------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n\n"
+             );
+
+  Table tableTest2( {"Duis fringilla, ligula sed porta fringilla,\nligula wisi commodo felis,ut adipiscing felis dui in enim. Suspendisse malesuada ultrices ante",
+                     "CordX",
+                     "CoordZ",
+                     "Prev\nelement",
+                     "Next\nelement"}
+                    );
+  tableTest2.setTitle( "InternalWellGenerator well_injector1" );
+  tableTest2.addRow( "value1", "[30.21543]", "3.0", 54, 0 );
+  tableTest2.addRow( "", "", "", "", "" );
+  tableTest2.addRow( "value23", "[30.21543]", "30.45465142", 787442, 10 );
+  tableTest2.draw( oss );
+  tableTestsOutput.push_back( oss.str() );
+  oss.clear();
+  oss.str( "" );
+  EXPECT_EQ( tableTestsOutput[1],
+             "\n+---------------------------------------------------------------------------------------------------------------------------------------------------------+\n"
+             "|                                                          InternalWellGenerator well_injector1                                                           |\n"
+             "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n"
+             "|                           Duis fringilla, ligula sed porta fringilla,                            |    CordX     |    CoordZ     |   Prev    |   Next    |\n"
+             "|  ligula wisi commodo felis,ut adipiscing felis dui in enim. Suspendisse malesuada ultrices ante  |              |               |  element  |  element  |\n"
+             "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n"
+             "|                                              value1                                              |  [30.21543]  |      3.0      |    54     |     0     |\n"
+             "|                                                                                                  |              |               |           |           |\n"
+             "|                                             value23                                              |  [30.21543]  |  30.45465142  |  787442   |    10     |\n"
+             "+--------------------------------------------------------------------------------------------------+--------------+---------------+-----------+-----------+\n\n"
+             );
```

Comment:
Consider surrounding your tests with braces to contain scope and get rid of the `2`, `3`, `4`.
Plus, if it makes sense, can you add a small line of comment to indicate what you're specifically testing?